1209/F16A/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Cougar TQS adapter
+owner: uri_ba
+license: MIT
```

Comment:
I don't see any indication of this license in your repository - could you add a license file that applies to your software and hardware, please?

Also, your hardware directory includes an Eagle schematic, but not a board file.